src/MSBuild.sln

```diff
@@ -607,44 +607,37 @@ Global
 		{11B5D53E-90E4-4BD5-9883-B5921F7DE854}.Release-NetCore|x64.Build.0 = Release|Any CPU
 		{11B5D53E-90E4-4BD5-9883-B5921F7DE854}.Release-NetCore|x86.ActiveCfg = Release|Any CPU
 		{11B5D53E-90E4-4BD5-9883-B5921F7DE854}.Release-NetCore|x86.Build.0 = Release|Any CPU
-		{53733ECF-0D81-43DA-B602-2AE9417F614F}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{53733ECF-0D81-43DA-B602-2AE9417F614F}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{53733ECF-0D81-43DA-B602-2AE9417F614F}.Debug|Any CPU.ActiveCfg = Debug|x86
```

Comment:
No I'm just using a newer version of VS so I thought it only updated the version in the SLN...